containers/python/test_sklearn_cifar_container.py

```diff
@@ -27,6 +30,7 @@ def load_cifar(cifar_location, cifar_filename = "train.data", norm=False):
         Z = Z.T
     return (Z, y)
 
+
```

Comment:
Only because it's part of the PEP 8 standard: https://www.python.org/dev/peps/pep-0008/#blank-lines